client/src/api.js

```diff
@@ -53,26 +55,28 @@ function addDrink(drinkObj) {
 }
 
 function getCookieData(sid) {
-  return axios.get(`/cookiedata`, {params: {sid: sid}})
-    .then(response => {
+  return axios
+    .get(`/cookiedata`, { params: { sid: sid } })
+    .then((response) => {
       return {
         user_id: response.data.user_id,
-        username: response.data.username
+        username: response.data.username,
       };
     })
-    .catch(err => console.log(err));
+    .catch((err) => console.log(err));
 }
 
 function getImage(shopObj) {
-  return axios.get('/getshopimage', {params: {shopObj} });
+  return axios.get('/getshopimage', { params: { shopObj } });
 }
 
-function likeShop(place_id, user_id) {
```

Comment:
This is the only function in this file that changed code-wise (no need to send user_id since the server already knows it, but added an option to use either post or delete path depending on intention)